docs/content/tools/deprecations.md

```diff
@@ -0,0 +1,91 @@
+## Deprecation data
```

Comment:
The `title` front matter is required for Doctocat to render this page correctly.

```suggestion
---
tite: Deprecation data
---
```